### ISSUE TITLE
Eric V.'s ballot review (6/25/2019)

### ISSUE THREAD

**Issue by csosto-pk** [csosto-pk/adding-shake-to-pkix#48]
== COMMENTS ==

-- Section 1 / Change log --

May I assume that the issues by the two reviews of -08 are solved in -11 ?

-- Section 4 --

== NITS ==

-- Abstract --

Just wondering why CRL acronym is expanded while SHAKE & ECDSA are not.

-- section 6 --

Also wondering why in some IANA entries "SHAKE" is in lower case while in others in upper case.

**Comment by csosto-pk** [comment]
> == COMMENTS ==
> 
> -- Section 1 / Change log --
> 
> May I assume that the issues by the two reviews of -08 are solved in -11 ?

Yes. And the upcoming update -12 will include all the fixes for the last 3 ballot position reviews. 

> -- Section 4 --
> 
> == NITS ==
> 
> -- Abstract --
> 
> Just wondering why CRL acronym is expanded while SHAKE & ECDSA are not.

Good point. Since we are in the abstract I kept CRL in there without expanding the acronym. But I made sure it is expanded in the first occurrence of the acronym in the Introduction section. We don't expand the SHAKE and ECDSA or RSAPSS because these are algorithm names, not exactly abbreviated acronyms. 

> -- section 6 --
> 
> Also wondering why in some IANA entries "SHAKE" is in lower case while in others in upper case.

The RSASSA-PSS OIDs kept SHAKE in capitals because it seemed unnatural to put it lowercase after the capitals of RSASSA-PSS. For ecdsa we kept it lowercase because that is how the OIDs looked in the past for ecdsa-sha2. It is a little arbitrary, but that is why we made them look like that.
